app/views/green_lanes/results/_result_category_2.html.erb

```diff
@@ -20,7 +20,7 @@
   <li>excise products and goods subject to special health, licensing or environmental controls</li>
 </ul>
 
-<%# <%= render 'green_lanes/shared/about_spimm_link' %> %>
+<%# <%= render 'green_lanes/shared/about_spimm_link' %>
```

Comment:
and this?

if you want to keep it, it's better to have just one open/close angular parenthesis <% %> 
`<%# = render 'green_lanes/shared/about_spimm_link' %>`